### ISSUE TITLE
AC129_c03468 has 2 bags: how to handle?

### ISSUE THREAD

**Issue by cwulfman** [pulibrary/webspace-to-figgy-transfer#6]
| collection code | component    | filepath                                                                                 | level of access | subdirs | files | viewables | 
|-----------------|--------------|------------------------------------------------------------------------------------------|-----------------|---------|-------|-----------| 
| AC129           | AC129_c03468 | /archives_bd/bcadmin/univarchives/AC129/AR.2009.129/AR_2009_129_bag_2/data/cdo           | open            | 2       | 0     | 0         | 
| AC129           | AC129_c03468 | /archives_bd/bcadmin/univarchives/AC129/AR.2009.129/AR_2009_129_bag/data/cdo/AR.2009.129 | open            | 3       | 0     | 0         | 

When ingested into Figgy as separate zip files, a search for AC129_c03468 returns two results, one for each zip, with the same component metadata.  Is this the desired behavior?

**Comment by aberdini** [comment]
@cwulfman That sounds like it should be okay to me, as long as there will still be a link for each zip and that those links don't wreck the PULFA user experience somehow -- we sometimes have to split content for the same accession or component into two bags in order for the bagging process to function correctly when there are very large files, so this won't be a unique situation. If that causes problems for access functionality, let me know, as that will require a larger workflow change on our end.

**Comment by cwulfman** [comment]
I'm afraid I'll need to defer to @escowles about links in the finding aid and what the user experience is going to be like with zip files.  Maybe I could toss the question back at you, actually: what do you think the user experience _should_ be when bags and zip files are involved?

**Comment by cwulfman** [comment]
I'm also a bit curious about the problem you're having with the bagging process requiring you to split content.  Is that a known limitation/bug with bagit?

**Comment by aberdini** [comment]
I think that our hope for the finding aid would be a nice digital content button that prompt a download, but of course that could look a little strange if there is more than one button in one component, so I'm not entirely sure how to handle that. 

As for the bagit problem -- it's one of memory limit as I understand. When we try to bag large volumes of content (generally anything over 500 GB at once) Bagger struggles to process the files and ends up stalling out and failing to bag -- but usually not for a day or so, and that time gets wasted. I believe this is a known problem with the tool.

**Comment by cwulfman** [comment]
These problems are not unrelated, are they? If the digital content for a particular component is numerous or large, the zip file might be burdensome for the user to download: AC122/c957 is 7.4 gigabytes, for example.  Should the extent indicate the size of the zip file?

**Comment by aberdini** [comment]
No, they are not unrelated, and yes, we are hoping to generate more useful extent information for our born-digital content. In the past, the digital files and folders count was used for extent, but one of our backburner projects is to fix that to reflect extent in GB. It's been on hold, mostly due to the Mudd move, but as content gets moved into Figgy we're going to work on fixing that issue.
